opae.spec.in

```diff
@@ -194,6 +196,8 @@ ldconfig
 %dir @CMAKE_INSTALL_PREFIX@/src/opae
 @CMAKE_INSTALL_PREFIX@/src/opae/*
 @CMAKE_INSTALL_PREFIX@/bin/opae.io
+@CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/libboard_common.a
```

Comment:
We were told not to package .a files.  It violates some sort of Fedora upstreaming guideline.

Reply:
You are right:
https://docs.fedoraproject.org/en-US/packaging-guidelines/#packaging-static-libraries

I've pushed new/updated commits where installation of the two static libraries is disabled instead.